src/pages/patterns/filters.mdx

```diff
@@ -0,0 +1,416 @@
+| Multiple selections, few categories of data              | A user wants to use multiple selections within different categories to quickly filter data                         | Nested menu                              |
+| Multiple selections, one category with many options      | A user wants to be able to search options and use multiple selections within a large set of options to filter data | Tooltip modal                            |
+| Many categories, live updates                            | A user wants to set multiple filters across a dataset with many categories and see their dataset update live       | Sheet                                    |
+| Many categories, batch updates                           | A user wants to set multiple filters across a dataset with many categories and apply them all at once              | Drawer                                   |
+
+## Usage
+
+### Basic filtering
+
+Basic Filters allow users to filter a dataset through one category with only a few options.
+
+#### Single select
+
+Single select basic filters use one selection to filter data. Upon the selection,
+the Menu closes and the filter is surfaced as a Tag.
+Selecting another menu option replaces the current Filter with the new selection.
+
+![Filter pattern basic single select 1](figma:filter-usage-basic-singleselect-1.png)
+![Filter pattern basic single select 2](figma:filter-usage-basic-singleselect-2.png)
+![Filter pattern basic single select 3](figma:filter-usage-basic-singleselect-3.png)
+![Filter pattern basic single select 4](figma:filter-usage-basic-singleselect-4.png)
+
+#### Multiple selections
+
+Multiple selection basic filters use multiple selections within a category to filter data.
+The Menu remains open as the user makes their selections.
+The selections are surfaced as Tags with a Basic Button to clear all filters.
+
+![Filter pattern basic multiple select 1](figma:filter-usage-basic-multipleselect-1.png)
+
+---
+
+![Filter pattern basic multiple select 2](figma:filter-usage-basic-multipleselect-2.png)
+
+#### Nested menu
+
+A [Nested menu](/components/menu) can be used to organize subcategories of data that can be used to filter a dataset.
+
+![Filter pattern basic nested menu 1](figma:filter-usage-basic-nestedmenu-1.png)
+
+---
+
+![Filter pattern basic nested menu 2](figma:filter-usage-basic-nestedmenu-2.png)
+
+---
+
+![Filter pattern basic nested menu 3](figma:filter-usage-basic-nestedmenu-3.png)
+
+### Advanced filtering
+
+Advanced Filters allow users to set and configure multiple filters on a dataset.
+
+#### Tooltip modal
+
+A Tooltip modal allows users to filter a dataset through one category with many options,
+and allows the user to search for specific options.
+They can also be part of a more complex flow.
+
+![Filter pattern advanced tooltip modal](figma:filter-usage-advanced-tooltip-modal.png)
```

Comment:
@eshet-hayil it just looks like the surrounding layer has too much padding. https://www.figma.com/file/HifWgSrdeTlMMXUrcnrAAH/Website-assets-(Figma)?type=design&node-id=2350-5799&mode=design&t=6PlumpEmCB6GtasZ-0. I suspect you can reduce to the size of the components and just let the paragraph spacing take care of the rest. @steue can you double-check for us?

Reply:
This image specifically has been a difficult one. Resizing it in Figma requires rebuilding it; is it possible to use a smaller PNG for this to reduce the size?

Reply:
Adjusted the image, should be smaller now